### ISSUE TITLE
Picker items sometimes not shown

### ISSUE THREAD

**Issue by cpetrov** [eclipsesource/tabris-js#1291]
### Problem description

- Actual Behavior: When a picker is appended to a page which is getting opened, its items are not shown. When tapping on the back button, items are rendered during the page close animation.
- Expected Behavior: Picker items should always appear on the second page from the provided snippet when it gets opened.

### Environment

- Device (name and OS version): iPhone 5s, iOS 10.3.1; S6, Android 7
- Tabris.js version: 2.x-nightly

### Code snippet

```javascript
const AIRPORTS = [
  'Airport 1',
  'Airport 2',
  'Airport 3'
];

let navigation = new tabris.NavigationView({left: 0, top: 0, right: 0, bottom: 0}).appendTo(tabris.ui.contentView);

new tabris.Page().appendTo(navigation).append(
  new tabris.Button({text: 'open page 2'}).on('select', () => {
    let page2 = new tabris.Page().appendTo(navigation);
    new tabris.Picker({
      left: 20, top: 20, right: 20,
      itemCount: AIRPORTS.length,
      itemText: index => AIRPORTS[index]
    }).appendTo(page2);
  })
);
```

**Comment by natanraj** [comment]
@ralfstx Any workaround on this ? I am seeing this issue in 2.0-rc1 build.

**Comment by cookieguru** [comment]
@natanraj This is fixed, but the fix was made after 2.0-rc1 was cut.  You can wait until 2.0-rc2 or use `2.x-nightly` (native side) and `nightly` (JS side)
